bin/http-server

```diff
@@ -18,41 +56,40 @@ if (argv.h || argv.help) {
     'usage: http-server [path] [options]',
     '',
     'options:',
-    '  -p           Port to use [8080]',
-    '  -a           Address to use [0.0.0.0]',
-    '  -d           Show directory listings [true]',
-    '  -i           Display autoIndex [true]',
-    '  -g --gzip    Serve gzip files when possible [false]',
-    '  -e --ext     Default file extension if none supplied [none]',
-    '  -s --silent  Suppress log messages from output',
-    '  --cors[=headers]   Enable CORS via the "Access-Control-Allow-Origin" header',
-    '                     Optionally provide CORS headers list separated by commas',
-    '  -o [path]    Open browser window after starting the server',
-    '  -c           Cache time (max-age) in seconds [3600], e.g. -c10 for 10 seconds.',
-    '               To disable caching, use -c-1.',
-    '  -U --utc     Use UTC time format in log messages.',
+    '  -p\t           Port to use [Default: 8080]',
```

Comment:
What are the `\t`'s for throughout this section?

Reply:
Those are tabs to make things line up better. In hind sight, this would be better handled by another library, or just not handled at all

Reply:
"This" being the alignment? I've done some work with http://yargs.js.org/ and really enjoyed it. Certainly less minimal...and would probably need full-on code restructuring...but if you're looking for something like this generally, yargs seems pretty fabulous.

Reply:
I did actually start out replacing optimist with yargs, but I went with minimist instead just because its so much smaller (minimist has no dependencies)

Reply:
I just replaced the tabs with spaces. I really think spaces are the way to go, I don't know what I was thinking before.